sky/policy.py

```diff
@@ -1,154 +1,82 @@
-"""Customize policy by users."""
-import copy
+"""Interface for admin-defined policy for user requests."""
+import abc
 import dataclasses
-import importlib
-import os
-import tempfile
 import typing
-from typing import Any, Callable, Dict, Optional
+from typing import Optional
 
-from sky import dag as dag_lib
-from sky import sky_logging
-from sky import skypilot_config
-from sky.utils import common_utils
-from sky.utils import ux_utils
+if typing.TYPE_CHECKING:
+    import sky
 
-logger = sky_logging.init_logger(__name__)
 
-if typing.TYPE_CHECKING:
-    from sky import task as task_lib
+@dataclasses.dataclass
+class OperationArgs:
```

Comment:
Naming: prefer "options" to "args", and "request" is more common in RPC. How about "RequestOptions"?

Reply:
Done.